web/gds-user-ui/src/modules/dashboard/member/MemberTable.tsx

```diff
@@ -114,7 +72,7 @@ const MemberTable: React.FC<MemberTableProps> = (data) => {
           </Tr>
         </Thead>
         <Tbody>
-          <TableRows />
+          <MemberTableRows rows={data} />
```

Comment:
With the addition of the `MemberTableRows` component, does this mean that the data will be passed down from the page component, to the table component, and then to the rows component?

Reply:
By adding this change, my goal is to separate the responsibilities of the `MemberTableRows` component, following the Single Responsibility principle from SOLID. The main purpose of this component is to retrieve data from either the props or the server state and display it. So yes for now the data will be passed down from the page component to the child that will consume the data (if we want to use the props, maybe the best option at this step). However, it's important to keep in mind that we'll need to review this implementation while working on the `Select Network` Logic